Registry.sql

```diff
@@ -0,0 +1,47 @@
+USE [WeddingSiteBuilder]
+GO
+
+/****** Object:  Table [dbo].[Registry]    Script Date: 12/3/2015 10:12:07 PM ******/
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
+SET ANSI_PADDING ON
+GO
+
+CREATE TABLE [dbo].[Registry](
+	[RegistryID] [bigint] IDENTITY(1,1) NOT NULL,
+	[WeddingID] [bigint] NOT NULL,
+	[StoreID] [bigint] NOT NULL,
```

Comment:
what is this store ID? wouldn't we just create a new registry and tie them all to one wedding?